sandesh/whatsapp.py

```diff
@@ -0,0 +1,24 @@
+import os
+from twilio.rest import Client
+
+class WhatsApp:
```

Comment:
can you run the code through pylint and flake8 please? it seems there are many spaces where its not needed. :)